doc/api/process.md

```diff
@@ -44,6 +44,32 @@ process.on('exit', (code) => {
   console.log('About to exit with code:', code);
 });
 ```
+## Event: 'exitingSoon'
+
+Emitted when `process.exit()` is called using the optional `timeout` argument.
```

Comment:
sorry no, in the refactored version this is changed. I just missed a doc edit... fixing now